test/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+const ERROR = "error";
```

Comment:
While we're making changes, I prefer using yaml for eslint personally. It's easier to read and you don't have to fuss with quotation marks and balancing braces. It also supports comments.